gradle/dependencies.gradle

```diff
@@ -17,26 +17,31 @@
 def versions = [
         androidTest: '0.5',
         androidxVersion: '1.0.0',
-        uberJava: '0.8.0',
+        uberJava: '0.8.1',
+]
+
+def network = [
+        moshi: 'com.squareup.moshi:moshi:1.9.2',
 ]
 
 def build = [
         gradleVersion: '4.9',
         buildToolsVersion: '28.0.2',
         compileSdkVersion: 28,
         ci: 'true' == System.getenv('CI'),
-        minSdkVersion: 15,
+        minSdkVersion: 16,
```

Comment:
On a separate PR, I would recommend bringing this up to a higher version to avoid a longtail of support issues. Minsdk 26 is probably reasonable here.

Reply:
sure